pkg/hubble/api/socket_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Authors of Hubble
```

Comment:
Shouldn't headers be updated to something like the following?
```
Copyright 2019 Authors of Hubble
Copyright 2020 Authors of Cilium
```

Reply:
Probably, this PR doesn't really change any code so I've just left copyright notices intact for now.